DXgovernance/davi-monorepo
# Feat: useGetActiveProposals: migrated hook to store

## Description

Migrated `useGetActiveProposals` hook to the store.

Also added `useListenToProposalStateChanged` event hook.

Closes #45

## Type of change

- [ ] Documentation (This change updates documenantion)
- [ ] Release (This pull request refers to a new version release)
- [ ] Other (This change updates documentation)

## How Has This Been Tested?

Manual testing and updated unit tests.

How to test:
 - Have two windows opened. In one, go to the main guild governance page, in the other, create a proposal.
 - You should see the new proposal in the first windows after creating it in the second window, without refreshing the page.

## Checklist:
